slack-invite.js

```diff
@@ -80,25 +80,151 @@ function slack_invite() {/*
+    <style>
+
+        body {
+            font-family: "Avenir Next",Avenir,"Helvetica Neue",Hevetica,sans-serif;
+        }
+
+        .header-image {
+            position: relative;
+            height: 235px;
+            width: 100%;
+            overflow: hidden;
+            margin-bottom: 70px;
+        }
+
+        .header-image .image{
+            background: transparent url("<%- ctx.secrets.BG_URL || ctx.secrets.LOGO_URL || logo_url %>") center center no-repeat;
+            background-size: cover;
+            -webkit-filter: blur(12px);
+            -moz-filter: blur(12px);
+            -o-filter: blur(12px);
+            -ms-filter: blur(12px);
+            filter: blur(12px);
+            position: absolute;
+            top: -20px;
+            right: -20px;
+            bottom: -20px;
+            left: -20px;
+            opacity: 0.9;
+        }
+
+        .user-logo {
+            width: 75px;
+            height: 75px;
+            border-radius: 50%;
+            box-shadow: 0px 2px 4px 0px #D0D2D3;
+            position: absolute;
+            top: 212px;
+            left: 49.5%;
+        }
+
+        .slack-logo {
+            width: 115px;
+            height: 115px;
+            border-radius: 50%;
+            box-shadow: 0px 2px 4px 0px #D0D2D3;
+            position: absolute;
+            top: 180px;
+            right: 48.5%;
+        }
+
+        p {
+            font-size: 20px;
+            margin: 15px 0 0; 
+        }
+
+        .input-email {
+            max-width: 325px;
+            margin: 40px auto;
+        }
+
+        .button-invitation {
+            background: #E76D5F;
+            border-radius: 25px;
+            font-size: 16px;
+            height: 50px;
+            border: none;
+            color: white;
+            padding: 0 30px;
+        }
+
+        .button-invitation:hover {
+            background: #DD6557;
+        }
+
+        .footer {
+            position: relative;
+            overflow: hidden;
+            margin: 40px 0;
+        }
+
+        .separator-line {
+            height: 1px;
+            width: 100%;
+            border-top: 1px solid #E76D5F;
+            position: absolute;
+            top: 50%;
+            left: 0;
+        }
+
+
+        .footer-copy {
+            background: white;
+        }
+
+        .webtask-copy a {
+            color: #333;
+        }
+
+        .webtask-copy a:hover {
+            text-decoration: none
+        }
+
+        .create-copy {
+            font-size: 16px;
+        }
+        
+        .create-copy a {
+            color: #E76D5F;
+        }
+
+        .create-copy a:hover {
+            color: #DD6557;
+        }
+
+    
+    </style>
+
 </head>
-<body style="padding-top: 30px;">
+<body>
+    <header>
+        <div class="header-image">
+            <div class="image"></div>
+        </div>
+        <img src="https://cdn.auth0.com/website/webtask/assets/slack-webtask-logo.png" alt="Slack" class="slack-logo">
```

Comment:
mmm no. We are promoting webtask on the footer. We want people to feel good about using it